docs/faq.md

```diff
@@ -2,27 +2,25 @@
 
 ## How do I know if watchmaker has installed?
 
-There are several ways to validate that watchmaker has been installed on your
-system. The simplest method is to run the command `watchmaker --help`. If it
-displays the cli help page, watchmaker is installed.
+To determine whether watchmaker is installed, the simplest method is to run the
+command `watchmaker --help`. If it displays the cli help page, watchmaker is
+installed. Another option is to check `pip list | grep watchmaker`.
 
 ## What do I do if watchmaker failed to install?
 
-First, review the installation documentation provided with the watchmaker
-package. If `pip` is not working as your primary installation process you can
-attempt to install the package by cloning the repo of watchmaker.
-
-```
-python setup.py install
-```
+First, review the [installation](installation.md) document. Then doublecheck
```

Comment:
The word is usually "double-check" with a dash in there, but I will admit that it transitioning to a single word might be a possibility by this point. :)